app/video_routes.py

```diff
@@ -0,0 +1,115 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from dotenv import load_dotenv
+load_dotenv()
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@videos_bp.route("", methods=["GET"])
+def get_video():
+    videos = Video.query.all()
+
+    response = []
+    for video in videos:
+        response.append(video.to_dict())
```

Comment:
List comprehension can be used for this line

```suggestion
    response = [ video.to_dict() for video in videos ]
```